src/gui/MailRecipientsTextField.ts

```diff
@@ -133,7 +135,12 @@ export class MailRecipientsTextField implements ClassComponent<MailRecipientsTex
 						}
 					} else {
 						return {
-							firstRow: "",
+							firstRow: m(Icon, {
```

Comment:
can we put a label like "Contact List" in the first row in addition to the icon?

Reply:
It was a bit noisy, do you think it's unclear?

Reply:
just thinking about accessibility, the icon won't be read by screenreaders right?

Reply:
ah that's a good point we should put the description somewhere